packages/cursorless-engine/src/processTargets/modifiers/scopeHandlers/CollectionItemScopeHandler/collectionItemIterationScopeHandler.ts

```diff
@@ -0,0 +1,24 @@
+import { testRegex } from "@cursorless/common";
+import type { TargetScope } from "../scope.types";
+import type { ComplexScopeType } from "../scopeHandler.types";
+import { separatorRegex } from "./getSeparatorOccurrences";
+
+export const collectionItemIterationScopeHandler: ComplexScopeType = {
+  type: "fallback",
+  scopeTypes: [
+    {
+      type: "surroundingPairInterior",
+      delimiter: "collectionBoundary",
+    },
```

Comment:
NTS: This will probably be the majority case because most lists are indeed inside of a surrounding pair somewhere